03/Product.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const buttonHandle = (props, e) => {
+    e.preventDefault();
+    const{buttonPress} = props;
+    const {id} = props;
+    buttonPress(id);
+    e.target.disabled = true;
```

Comment:
Zmiana tego w ten sposób nie jest najlepszym pomysłem.
Lepiej taką informacje przekazać przez `props` - zobacz, że jak dodasz produkt (kupisz), a potem usuniesz to `disabled` "nie wraca" do odpowiedniego stanu.